docs/pages/guides/authentication.mdx

```diff
@@ -72,6 +72,8 @@ If you'd like to see more, you can [open a PR](https://github.com/expo/expo/edit
   <GridItem title="Twitch" protocol={['OAuth 2']} image={ASSETS.twitch} />
   <GridItem title="Twitter" protocol={['OAuth 2']} image={ASSETS.twitter} />
   <GridItem title="Uber" protocol={['OAuth 2']} image={ASSETS.uber} />
+  <GridItem title="Calendly" protocol={['OAuth 2']} image={ASSETS.calendly} />
```

Comment:
Let's not add this at the end of this Grid. On this page, we follow alphabetical order so let' stick to that. Please remove this line from here and add it before "Cognito".